cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -1706,43 +1706,46 @@ impl_runtime_apis! {
 				Config as XcmBridgeHubRouterConfig,
 			};
 
+			use testnet_parachains_constants::rococo::locations::{AssetHubParaId, AssetHubLocation};
 			parameter_types! {
 				pub ExistentialDepositAsset: Option<Asset> = Some((
 					TokenLocation::get(),
 					ExistentialDeposit::get()
 				).into());
-				pub const RandomParaId: ParaId = ParaId::new(43211234);
+
+				pub RandomParaId: ParaId = ParaId::new(43211234);
 			}
 
 			use pallet_xcm::benchmarking::Pallet as PalletXcmExtrinsicsBenchmark;
 			impl pallet_xcm::benchmarking::Config for Runtime {
 				type DeliveryHelper = (
-					cumulus_primitives_utility::ToParentDeliveryHelper<
+				polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
 						xcm_config::XcmConfig,
 						ExistentialDepositAsset,
-						xcm_config::PriceForParentDelivery,
+						PriceForSiblingParachainDelivery,
+						RandomParaId,
+						ParachainSystem
 					>,
-					polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
+				polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
 						xcm_config::XcmConfig,
 						ExistentialDepositAsset,
 						PriceForSiblingParachainDelivery,
-						RandomParaId,
-						ParachainSystem,
-					>
-				);
+						AssetHubParaId,
+						ParachainSystem
+					>);
 
 				fn reachable_dest() -> Option<Location> {
-					Some(Parent.into())
+					Some(AssetHubLocation::get())
 				}
 
 				fn teleportable_asset_and_dest() -> Option<(Asset, Location)> {
 					// Relay/native token can be teleported between AH and Relay.
 					Some((
 						Asset {
 							fun: Fungible(ExistentialDeposit::get()),
-							id: AssetId(Parent.into())
+							id: AssetId(TokenLocation::get())
 						},
-						Parent.into(),
+						AssetHubLocation::get(),
```

Comment:
@karolk91 @EleisonC also just for AssetHubs, here it should be `RandomParaLocation` (probably won't work because of `IsTeleport`), or maybe use some system parachain, e.g. CoretimeParaId - because we can teleport between system paras:
```suggestion
						CoretimeLocation::get(),
```

Reply:
Maybe we don't need to add `CoretimeParaId` at all, if everything works with customizing for benchmarking:
```
type TrustedTeleporter = TrustedTeleporter;
type TrustedReserve = TrustedReserve;
```